app/services/data/insert-workload-points.js

```diff
@@ -0,0 +1,43 @@
+const config = require('../../../knexfile').web
+const knex = require('knex')(config)
+
+module.exports = function (workloadPoints) {
+  return knex('workload_points')
+    .insert({
+      comm_tier_1: workloadPoints.commA,
+      comm_tier_2: workloadPoints.commB1,
+      comm_tier_3: workloadPoints.commB2,
+      comm_tier_4: workloadPoints.commC1,
+      comm_tier_5: workloadPoints.commC2,
+      comm_tier_6: workloadPoints.commD1,
+      comm_tier_7: workloadPoints.commD2,
+      cust_tier_1: workloadPoints.cusA,
+      cust_tier_2: workloadPoints.cusB1,
+      cust_tier_3: workloadPoints.cusB2,
+      cust_tier_4: workloadPoints.cusC1,
+      cust_tier_5: workloadPoints.cusC2,
+      cust_tier_6: workloadPoints.cusD1,
+      cust_tier_7: workloadPoints.cusD2,
+      lic_tier_1: workloadPoints.licA,
+      lic_tier_2: workloadPoints.licB1,
+      lic_tier_3: workloadPoints.licB2,
+      lic_tier_4: workloadPoints.licC1,
+      lic_tier_5: workloadPoints.licC2,
+      lic_tier_6: workloadPoints.licD1,
+      lic_tier_7: workloadPoints.licD2,
+      // TODO: Insert the actual id of the logged in user who changed the points
```

Comment:
Has this follow-up action been noted anywhere else - e.g. specific story in Taiga?

Reply:
Ticket created in this sprint on Tiaga